mono/metadata/metadata-internals.h

```diff
@@ -757,15 +757,15 @@ mono_metadata_interfaces_from_typedef_full  (MonoImage             *image,
 											 MonoGenericContext    *context,
 											 MonoError *error);
 
-MONO_API MonoMethodSignature *
+MonoMethodSignature *
```

Comment:
Why this change?

Reply:
I thought it good to remove these in general if xamarin doesn't use them and not in public api. esp. if they take MonoError. Easier to audit..